src/pages/my-patches/PatchCard.tsx

```diff
@@ -0,0 +1,128 @@
+import React, { useEffect } from "react";
+import styled from "@emotion/styled";
+import { PatchStatusBadge } from "components/PatchStatusBadge";
+import { GET_PATCH_VARIANTS_AND_STATUS } from "gql/queries/my-patches";
+import { BuildStatusIcon } from "pages/my-patches/patch-card/BuildStatusIcon";
+import { uiColors } from "@leafygreen-ui/palette";
+import Button from "@leafygreen-ui/button";
+import { format } from "date-fns";
+import { StyledLink } from "components/styles";
+import { paths } from "constants/routes";
+import { useQuery } from "@apollo/react-hooks";
+import get from "lodash/get";
+import {
+  PatchBuildVariantsAndStatusQueryVariables,
+  PatchBuildVariantsAndStatusQuery,
+  Maybe,
+} from "gql/generated/types";
+
+interface Build {
+  buildVariant: string;
+  status: string;
+}
+interface Props {
+  id: string;
+  projectID: string;
+  description: string;
+  status: string;
+  createTime?: Maybe<Date>;
+  builds: Build[];
+}
+export const PatchCard: React.FC<Props> = ({
+  id,
+  description,
+  createTime,
+  projectID,
+  ...props
+}) => {
+  const { data, stopPolling } = useQuery<
+    PatchBuildVariantsAndStatusQuery,
+    PatchBuildVariantsAndStatusQueryVariables
+  >(GET_PATCH_VARIANTS_AND_STATUS, {
+    variables: { id },
+    pollInterval: 5000,
+  });
+  useEffect(() => stopPolling, [stopPolling]);
+  const status: string = get(data, "patch.status", props.status);
+  const builds: Build[] = get(data, "patch.builds", props.builds);
+
+  const createDate = new Date(createTime);
+  return (
+    <CardWrapper data-cy="patch-card">
+      <Left>
+        <DescriptionLink href={`${paths.patch}/${id}`}>
+          {description || "no description"}
+        </DescriptionLink>
+        <TimeAndProject>
+          {format(createDate, "M/d/yy")} at {format(createDate, "h:mm:ss aaaa")}{" "}
+          on <b>{projectID}</b>
+        </TimeAndProject>
+      </Left>
+      <Center>
+        <BadgeContainer>
+          <PatchStatusBadge status={status} />
+        </BadgeContainer>
+        <IconsContainer>
+          {builds.map((b, i) => (
+            <div key={i}>
```

Comment:
Its not generally a good  practice to use indexes as keys. https://reactjs.org/docs/lists-and-keys.html#keys 
Try using the build id or some other unique field.

Reply:
good call